build/ublue-os-just/libjust.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+# shellcheck source=/dev/null
+# Import color formatting
+source /usr/lib/ujust/libcolors.sh
+# Import text formatting
+source /usr/lib/ujust/libformatting.sh
+# Import functionality for just
+source /usr/lib/ujust/libfunctions.sh
```

Comment:
Minor suggestion for future maintenance and improved organization of the bash lib files.

What made me start thinking about this: it's slightly inconsistent that we have `/usr/lib/ujust` but the main file is `libjust.sh` not `libujust.sh` or `ujust.sh`.

So one suggestion...

Within the source `ublue-os-just`:
- make a subdir `lib-ujust/`
- move all the `lib*.sh` files into the subdir, which keeps them clearly separated from other scripts which may end up in that source dir
- rename `libjust.sh` to `ujust.sh`
- the end result for `/usr/lib/ujust` on a users filesystem will be the same as this code already will install, but with `/usr/lib/ujust/ujust.sh` as a clear entrypoint which includes the other `lib*.sh` files there.

Reply:
this is what feedback is good for since i was not entirely sure myself what to name it so i went with `libjust` (or `justlib` for a bit) so that i had something to at least test with until things got ready. i will however move the files and change `libjust.sh` to `ujust.sh` so that is done before merge.

Reply:
If I missed some other inconsistency, I'd approach it the same way... You've built `libujust` which will live at `/usr/lib/ujust` ... I think the `ujust` distinction is important since it's heavily tied to the `ujust` implementation and won't impact standard `just` usages (unless a user very specifically searched it out and imported it in their Justfile`.